src/main/java/com/sucy/skill/listener/ExperienceListener.java

```diff
@@ -85,6 +89,24 @@ public void onCraft(CraftItemEvent event) {
             }
         }
     }
+    @EventHandler(priority = EventPriority.HIGHEST, ignoreCancelled = true)
+    public void onExpGain(PlayerExperienceGainEvent event){
+        Player player = event.getPlayerData().getPlayer();
+        Set<PermissionAttachmentInfo> perms = player.getEffectivePermissions();
+        OptionalInt max = perms.stream()
+                .filter(c->c.getPermission().startsWith("skillapi.exp.booster"))
+                .map(c->c.getPermission().substring(21))
+                .mapToInt(number->{
+                    try{
+                        return Integer.parseInt(number);
+                    }catch (NumberFormatException e){
+                        return 0;
+                    }
+                })
+                .max();
+        if (max.isEmpty()) return;
+        event.setExp(event.getExp() + event.getExp()*max.getAsInt()/100);
```

Comment:
Base value would have to be a booster value of 100 in order to not affect anything? I guess this would be decent to allow nerfing exp for some players, or boosting it for others.

Reply:
`skillapi.exp.booster.100` now will be 1.0x exp modifier, 50 will be 0.5x mod
Though I didn't test this commit specifically, but last one worked so this one should too